examples/custom-map-style/src/store.ts

```diff
@@ -18,8 +18,10 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-import {createStore, combineReducers, applyMiddleware, compose} from 'redux';
+import {combineReducers, applyMiddleware} from 'redux';
 import keplerGlReducer, {enhanceReduxMiddleware} from '@kepler.gl/reducers';
+import { configureStore } from '@reduxjs/toolkit';
+import { StoreEnhancer } from 'redux';
```

Comment:
@macrigiuseppe Need your review on these redux changes.

Reply:
for context - createStore had a deprecation warning, and a IDE note to use configureStore. It's another thing that can be pulled to another PR if it's easier.

Reply:
<img width="587" alt="Screenshot 2023-11-30 at 22 14 44" src="https://github.com/keplergl/kepler.gl/assets/74932975/dec3c014-5dae-4944-b86e-cfabe63383be">